### PR TITLE
Feat/add sign up student

### PR DESCRIPTION
**Pull request by medson10**

### CODE REVIEW

**Comment by medson10** [comment]
[Componente Concluir Cadastro Aluno](https://trello.com/c/bWj2cS2b/48-componente-concluir-cadastro-aluno)

**Review by Vitormdias — CHANGES_REQUESTED** [review]
Conclude -> Finish

File: src/router/index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import Vue from 'vue'
 import Router from 'vue-router'
 import Home from '@/components/Home'
+import ConcludeSignUp from '@/components/ConcludeSignUp'
```
Comment (Vitormdias) [inline src/router/index.js]: here

File: src/router/index.js [hunk 2]
```diff
@@ -9,5 +10,10 @@ export default new Router({
     path: '/',
     name: 'Home',
     component: Home
+  },
+  {
+    path: '/signup',
+    name: 'ConcludeSignUp',
```
Comment (Vitormdias) [inline src/router/index.js]: here

File: src/router/index.js [hunk 3]
```diff
@@ -9,5 +10,10 @@ export default new Router({
...
+    component: ConcludeSignUp
```
Comment (Vitormdias) [inline src/router/index.js]: here

**Review by Vitormdias — CHANGES_REQUESTED** [review]

**Review by Vitormdias — APPROVED** [review]

**Pull request merged**